### PR TITLE
Use reflection to resolve protobuf message types

### PR DESCRIPTION
**Pull request by changlan**
This PR avoids manually enumerating message types as we current do.

### CODE REVIEW

File: libbess-python/bess.py [hunk 1]
```diff
@@ -199,14 +200,15 @@ def create_port(self, driver, name, arg):
...
         request = proto_conv.dict_to_protobuf(kv, bess_msg.CreatePortRequest)
-        message_map = {
-            'PCAPPort': port_msg.PCAPPortArg,
-            'PMDPort': port_msg.PMDPortArg,
-            'UnixSocketPort': port_msg.UnixSocketPortArg,
-            'ZeroCopyVPort': port_msg.ZeroCopyVPortArg,
-            'VPort': port_msg.VPortArg,
-        }
-        arg_msg = proto_conv.dict_to_protobuf(arg, message_map[driver])
+        all_classes = inspect.getmembers(port_msg,
```
Comment (sangjinhan) [inline libbess-python/bess.py]: Perhaps something like this is more intuitive:
```
message_type = getattr(port_msg, driver + 'Arg', bess_msg.EmptyArg)
arg_msg = proto_conv.dict_to_protobuf(arg, message_type)
```

File: libbess-python/bess.py [hunk 2]
```diff
@@ -241,54 +243,15 @@ def create_module(self, mclass, name=None, arg=None):
...
-            'UrlFilter': module_msg.UrlFilterArg,
-            'VLANPop': bess_msg.EmptyArg,
-            'VLANPush': module_msg.VLANPushArg,
-            'VLANSplit': bess_msg.EmptyArg,
-            'VXLANDecap': bess_msg.EmptyArg,
-            'VXLANEncap': module_msg.VXLANEncapArg,
-            'WildcardMatch': module_msg.WildcardMatchArg,
-        }
-        arg_msg = proto_conv.dict_to_protobuf(arg, message_map[mclass])
+        all_classes = inspect.getmembers(module_msg,
```
Comment (sangjinhan) [inline libbess-python/bess.py]: same

File: .gitignore [hunk 3]
```diff
@@ -47,6 +47,7 @@ deps/*.gz
 *.pb.cc
 *.pb.h
 *_pb2.py
+*_pb2_grpc.py
```
Comment (sangjinhan) [inline .gitignore]: Do we ever generate this type of files?

Comment (changlan) [inline .gitignore]: I do have these files. Perhaps it is the newer version of grpc/protobuf?

**Comment by sangjinhan** [comment]
Thanks!

**Pull request merged**